packages/tupaia-web/src/views/ProjectPage/ProjectPage.tsx

```diff
@@ -1,19 +1,44 @@
-/*
+/**
  * Tupaia
- *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
+
 import React from 'react';
-import { DesktopLayout } from './DesktopLayout';
+import styled from 'styled-components';
+import { EnvBanner } from '@tupaia/ui-components';
+import { TopBar } from '../../layout';
+import { MOBILE_BREAKPOINT } from '../../constants';
 import { MobileTabLayout } from './MobileTabLayout';
+import { DesktopLayout } from './DesktopLayout';
 
 /**
- * This is the layout for the project/* view. This contains the map and the sidebar, as well as any overlays that are not auth overlays (i.e. not needed in landing pages)
+ * This is the layout for the entire app, which contains the top bar and the main content. This is used to wrap the entire app content
  */
```

Comment:
I'm slightly tempted to rename this file to something else, but I'm also not sure. Just because `ProjectPage` implies the `projectCode` route, since that's the sort of pattern we have going, and it's not strictly that route, if that makes sense. But I am also on the fence about this one.

Reply:
I know what you mean. I actually just added back the MainLayout and wrapped that around the routes at the top level and so now I think it makes a bit more sense to be called project page again.